src/components/KFileUpload/KFileUpload.vue

```diff
@@ -26,9 +30,9 @@
       </span>
 
       <KInput
+        v-bind="attrs"
```

Comment:
The [sandbox preview deploy](https://deploy-preview-2241--kongponents-sandbox.netlify.app/#/fileupload) is broken because when we set `model-value` on a KFileUpload, it'll be passed to the `<input>` element. I suggest changing this line to:
```suggestion
        v-bind="attrs.id ? { id: String(attrs.id) } : {}"
```

Reply:
Done